### ISSUE TITLE
Simulation Error, extdata found 1 unfulfilled imports

### ISSUE THREAD

**Issue by beiduqiu** [geoschem/geos-chem#2165]
### Name and Institution (Required)

Name: Zifan WANG
Institution: Washington University in St. Louis

### Confirm you have reviewed the following documentation

- [x] [Support guidelines](https://geos-chem.readthedocs.io/en/stable/help-and-reference/SUPPORT.html)
- [x] [User manuals](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/related-docs.html)
- [x] [Debugging GEOS-Chem and HEMCO errors](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
- [x] [Current and past Github issues](https://github.com/geoschem/geos-chem/issues)

### Description of your issue or question
When I update the GCHP from version 14.1.0 to 14.3.0, I met the following issue while running the simulation.
![image](https://github.com/geoschem/geos-chem/assets/86814483/9eb96630-a454-487d-9a26-40e7dfbee41d)
However, when I try to comment the correponding line or changing the error message and recompile it to find more detailed information. It still shows the same information. I recompile the program by deleting the build file and recompile it by following the GCHP quick start guide. May I ask which module I am missing and is this the correct way to recompile the whole project? Thanks in advance for your help.
![image](https://github.com/geoschem/geos-chem/assets/86814483/3ee95bc9-a0f5-4dde-a3d3-35130983a7cc)

Please provide as much detail as possible. Always include the GEOS-Chem version number and any relevant configuration and log files.

**Comment by lizziel** [comment]
Hi @beiduqiu, there is an additional log file in GCHP called `allPEs.log` that contains MAPL logging information, including ExtData error messages. The error you are encountering is because one of the import files is not found, perhaps because it is new in 14.3 and you have not downloaded the data. Check the `allPEs.log` file to see if there is a message about which import is missing. 

For more information about MAPL logging, GCHP log files, and debugging, see the following resources:

- https://gchp.readthedocs.io/en/stable/user-guide/debugging.html
- https://gchp.readthedocs.io/en/stable/user-guide/output_files.html#file-descriptions
- https://gchp.readthedocs.io/en/stable/user-guide/config-files/logging_yml.html

**Comment by yantosca** [comment]
@lizziel: should we transfer this issue to the GCHP repo?

**Comment by lizziel** [comment]
@yantosca, yes I'll do that. I didn't notice it wasn't under the GCHP repo issues.
